Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -70,10 +310,18 @@ def main():
         client.set_credentials(api_user, api_key)
 
         if command == 'test-module':
```

Comment:
Why have this extra condition instead of just adding `test-module` to the commands map below?

Reply:
As test module has no arguments, and the below map is created for commands having arguments.

Reply:
Ok.